fields/types/relationship/RelationshipType.js

```diff
@@ -221,11 +221,6 @@ relationship.prototype.updateItem = function (item, data, callback) {
 		throw new Error('fieldTypes.relationship.updateItem() Error - You cannot update populated relationships.');
 	}
 
-	var value = this.getValueFromData(data);
```

Comment:
This line needs to stay or there is an error `ReferenceError: value is not defined`